### DISCUSSION TITLE
How to deal with this issue in StackStorm HA https://github.com/StackStorm/st2/issues/4716 ?

### DISCUSSION THREAD

**Discussion by bell-manz** [StackStorm/st2#6000] [Operations] [answered]
Hi, my team at my company is trying to deploy the https://github.com/StackStorm/stackstorm-k8s charts in OpenShift. We successfully de-rooted the entire project and deployed it in OpenShift and everything is working, but we are experiencing this issue https://github.com/StackStorm/st2/issues/4716 which isn't acceptable for our use case. How are you guys getting around this issue in your StackStorm deployment? Any ideas? Also, does this issue actually affect anything in StackStorm besides the UI saying that the action is running forever? Thanks. :)

**Comment by arm4b** [comment, marked as answer]
Looks like the issue was answered here: https://github.com/StackStorm/st2/issues/4716
Did that solve it for you @bell-manz ?

**Comment by guzzijones** [comment]
More [graceful shutdown gotchas](https://github.com/StackStorm/stackstorm-k8s/issues/381)
